src/commands/cmd_stream.cc

```diff
@@ -242,6 +242,100 @@ class CommandXDel : public Commander {
   std::vector<redis::StreamEntryID> ids_;
 };
 
+class CommandXClaim : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() < 6) {
+      return {Status::RedisParseErr, errWrongNumOfArguments};
+    }
+
+    CommandParser parser(args, 1);
+    stream_name_ = GET_OR_RET(parser.TakeStr());
+    group_name_ = GET_OR_RET(parser.TakeStr());
+    consumer_name_ = GET_OR_RET(parser.TakeStr());
+    min_idle_time_ = GET_OR_RET(parser.TakeInt<uint64_t>());
```

Comment:
Here should be int64_t, and if the min_idle_time < 0, we need to reset it to 0.

Reply:
I see I see, just wondering if that's the case, for `idle`, `time` and `retrycount`, are they all going to be int64_t, followed by some parse result checking?

Reply:
Yes, I believe it's the same way Redis uses.